intel_extension_for_transformers/backends/neural_engine/graph/models/llama/llama.cpp

```diff
@@ -138,13 +168,19 @@ static bool llama_model_eval_internal(model_context& lctx, const model_token* to
       Kcur = ne_rope_inplace(
           ctx0, ne_reshape_3d(ctx0, ne_mul_mat(ctx0, model.layers[il].attn[1], cur), n_embd / n_head, n_head, N),
           n_past, n_rot, 0);
-      Vcur = ne_transpose(ctx0, ne_reshape_2d(ctx0, ne_mul_mat(ctx0, model.layers[il].attn[2], cur), n_embd, N));
+      if (!kv_mem_jblas) {
+        Vcur = ne_transpose(ctx0, ne_reshape_2d(ctx0, ne_mul_mat(ctx0, model.layers[il].attn[2], cur), n_embd, N));
+      } else {
+        Vcur = ne_rope_inplace(
```

Comment:
`Vcur` does not need `rope_embedding`

Reply:
Thanks. It has been fixed in 90e51f6 and the performance data in the PR description has been updated accordingly.